src/Command/InfectionCommand.php

```diff
@@ -159,8 +162,7 @@ protected function configure(): void
                 'coverage',
                 null,
                 InputOption::VALUE_REQUIRED,
-                'Path to existing coverage (`xml` and `junit` reports are required)',
-                ''
```

Comment:
There is absolutely no benefit to have the default as an empty string so I removed it